.gitignore

```diff
@@ -11,4 +11,4 @@ build/*
 *.egg-info/
 *.egg
 .eggs/
-
+.python-version
```

Comment:
I'm wondering what are the best practices on check-in vs ignore this file... do you have an opinion?

Reply:
I say ignore, since `pyenv local` will create it.

Reply:
that's a fair point, and the README is already clear enough on the supported versions and how to set them up using pyenv

Reply:
ok, It remains git ignored. Thanks both.